Dockerfile

```diff
@@ -94,11 +86,7 @@ COPY ./LICENSE /workspace/bionemo2/LICENSE
 COPY ./3rdparty /workspace/bionemo2/3rdparty
 COPY ./sub-packages /workspace/bionemo2/sub-packages
 
-# Note, we need to mount the .git folder here so that setuptools-scm is able to fetch git tag for version.
```

Comment:
we don't use setuptools-scm, and doing this requires a full git fetch rather than a shallow clone, so we remove mounting the `.git` folder